br/pkg/lightning/importer/get_pre_info.go

```diff
@@ -545,26 +556,35 @@ func (p *PreImportInfoGetterImpl) EstimateSourceDataSize(ctx context.Context, op
 					tbl.IndexRatio = sampledIndexRatio
 					tbl.IsRowOrdered = isRowOrderedFromSample
 
+					// if sample data failed due to max-error, fallback to use source size
 					if tbl.IndexRatio > 0 {
-						sizeWithIndex += int64(float64(tbl.TotalSize) * tbl.IndexRatio)
-					} else {
-						// if sample data failed due to max-error, fallback to use source size
-						sizeWithIndex += tbl.TotalSize
+						tableSize = int64(float64(tbl.TotalSize) * tbl.IndexRatio)
```

Comment:
Can we remove the `if tbl.IndexRatio > 0`?  It seems `tbl.IndexRatio` can not be zero.

Reply:
It is removed in https://github.com/pingcap/tidb/pull/41943/commits/931f006043591ae0ec7cb22e190013570b15cfad